Source/main.cpp

```diff
@@ -1,10 +1,78 @@
 #include <iostream>
-#include <string>
 #include "brute_force.h"
 #include "ciphers.h"
 
-int main()
-{
+int main() {
+    std::string password;
+    std::string encryptedWord;
+
+    // User inputs password
+    std::cout << "Enter the password: ";
+    std::cin >> password;
+
+    // Choose encryption method (Caesar or Vigenere)
+    int choice;
+    std::cout << "Select encryption method: \n";
+    std::cout << "1. Caesar\n2. Vigenere\n";
+    std::cin >> choice;
+
```

Comment:
Need a STL or API clock timer for how long encryption takes that prints after the encryption.